src/inference/dev_api/ie_system_conf.h

```diff
@@ -176,4 +189,27 @@ typedef enum {
     CPU_MAP_USED_FLAG = 5      //!< column for resource management of the processor
 } column_of_cpu_mapping_table;
 
+#if (defined(_WIN32) || defined(_WIN64))
+/**
+ * @brief      Parse processors infomation on Windows
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  base_ptr buffer object pointer of Windows system infomation
+ * @param[in]  len buffer object length of Windows system infomation
+ * @param[in]  _processors total number for processors in system.
+ * @param[out] _sockets total number for sockets in system
+ * @param[out] _cores total number for physical CPU cores in system
+ * @param[out] _proc_type_table summary table of number of processors per type
+ * @param[out] _cpu_mapping_table CPU mapping table for each processor
+ * @return
+ */
+INFERENCE_ENGINE_API_CPP(void)
+parse_processor_info_win(const char* base_ptr,
+                         const unsigned long len,
+                         const int _processors,
+                         int& _sockets,
+                         int& _cores,
+                         std::vector<std::vector<int>>& _proc_type_table,
+                         std::vector<std::vector<int>>& _cpu_mapping_table);
```

Comment:
Just one remaining thought. Does it make sense to incapsulate both `_proc_type_table` and `_cpu_mapping_table` inside one class and give read only API for its content? Both tables are initialized once and have to be consistent at any time of programm execution. Besides that the only field that can be changes after tables initialization is `Used`  in `_cpu_mapping_table`. so after fileds have to be protected from modification. Given that using Class seems to be most suitable solution here.